src/js/actions/layers.js

```diff
@@ -1076,7 +1076,7 @@ define(function (require, exports) {
         }
 
         // If document doesn't exist, or is a flat document
-        if (!document || document.unsupported || document.layers.all.size === 1 &&
```

Comment:
I removed the `document.unsupported` checker here for deselecting all layers for unsupported document. I read the code where `deselectAll` and looks like it is safe.